apps/frontend/src/libs/components/page-layout/styles.module.css

```diff
@@ -0,0 +1,41 @@
+:root {
+	--header-height: 73px;
+}
```

Comment:
Look at [this comment](https://github.com/BinaryStudioAcademy/bsa-2024-gitfit/pull/105#discussion_r1734270942)